pkg/engine/runtime/terraform/util.go

```diff
@@ -0,0 +1,163 @@
+package terraform
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"time"
+
+	"github.com/hashicorp/hc-install/product"
+	"github.com/hashicorp/hc-install/releases"
+	apiv1 "kusionstack.io/kusion/pkg/apis/core/v1"
+	"kusionstack.io/kusion/pkg/util/kfile"
+	"kusionstack.io/kusion/pkg/util/pretty"
+)
+
+var (
+	tfInstallSubDir  = "terraform"
+	tfInstallTimeout = 3 * time.Minute
+)
+
+type InstallOption struct {
```

Comment:
I would suggest rename `InstallOption` to `CLIInstaller`, also rename `util.go` to `installer.go`